__tests__/app.test.js

```diff
@@ -111,6 +111,59 @@ describe("GET /api/articles/:id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: returns all comments related to article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.comments).toBeInstanceOf(Array);
+        res.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            article_id: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("200: returns all comments when article only has one comment", () => {
+    return request(app)
+      .get("/api/articles/6/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.comments).toBeInstanceOf(Array);
+        res.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            article_id: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("200: this article has no comments", () => {
+    return request(app)
+      .get("/api/articles/4/comments")
+      .expect(200)
```

Comment:
A valid article with no comments shouldn't return a 200... have a think about which status code you need for "no content" and implement that instead!